pennylane/ops/qubit.py

```diff
@@ -2505,6 +2505,92 @@ def diagonalizing_gates(self):
         """
         return [QubitUnitary(self.eigendecomposition["eigvec"].conj().T, wires=list(self.wires))]
 
+# =============================================================================
+# Arithmetic
+# =============================================================================
+
+class Carry(Operation):
+    r"""Carry()
+    Apply a ``Carry`` operation on input wires
+    TODO add more information
```

Comment:
A diagram would be useful, so that it's clear what each wire corresponds to

Reply:
I'd like to do this. Where is the best place to include the diagram? In the docs? Or is there some way to include it here?